cmd/subctl/verify.go

```diff
@@ -142,6 +143,8 @@ func addVerifyFlags(cmd *cobra.Command) {
 		"namespace in which submariner is deployed")
 	cmd.Flags().StringVar(&verifyOnly, "only", strings.Join(getAllVerifyKeys(), ","), "comma separated verifications to be performed")
 	cmd.Flags().BoolVar(&disruptiveTests, "disruptive-tests", false, "enable disruptive verifications like gateway-failover")
+	cmd.Flags().UintVar(&packetSize, "packet-size", 3000, "size of packet")
```

Comment:
Is the default packet-size of 3000 matching the current implementation?

Reply:
Correct, with size equals to 3000 we'll echo 50 times [1]  data buffer which is similar to the current status.


[1] 
https://github.com/submariner-io/shipyard/pull/1140/files#diff-8e8c299993caac0264030c88915843d710ec8f4ef455082beb1683e2b9cbc27cR276

Reply:
AFAICT there is no specific reason behind 3000 bytes. Reducing the payload may not really help in shortening the test time when the connections are good. The time difference between 3000 bytes vs 1600 bytes may not be significant, the real difference will be when there is connection failure and the values of `--connection-attempts` and `--connection-timeout` come into picture. Sometimes, while testing, I use custom values for these parameters to shorten the time interval.

Reply:
Can we provide more help description than just "size of packet" ?